docs/guide/field_type_reference.md

```diff
@@ -1,12 +1,15 @@
 # Field Types reference
 
+A Field Type is the underlying building block of a content model. It consists of two entities: Field value and Field definition. Field value is determined by values entered into the Content Field. Field definition is provided by the Content Type, and holds any user defined rules used by Field Type to determine how a Field Value is validated, stored, retrieved, formatted and so on.
```

Comment:
Maybe wrong so more for @DominikaK to review on, but should it be "the content model" as we refer to our one and only amazing content model ;)

Reply:
Agreed :)